content/resources/lessons/sound-basics/presentation/index.md

```diff
@@ -91,6 +102,15 @@ $$
 
 {{% /fragment %}}
 
+>Notes:
+>- Decibels are always the log of a ratio.
+>- We typically scale based on a pain threshold of 20 Pa, and a reference value of
```

Comment:
Do you really mean pain? Perhaps gain? Otherwise, a reference please?

Reply:
I have now removed this part, I think I misunderstood what I was reading.